src/guacenc/ffmpeg-compat.c

```diff
@@ -165,3 +197,62 @@ int guacenc_avcodec_encode_video(guacenc_video* video, AVFrame* frame) {
 #endif
 }
 
+AVCodecContext* guacenc_build_avcodeccontext(AVStream* stream,
+        AVCodec* codec,
```

Comment:
Please follow the general style of the surrounding code. This function stands out as the only one to dedicate a separate line to each parameter, regardless of line length.

Reply:
My mistake! I write Java all day at work and so the only excuse I can think of for why I didn't see this when I was eyeball-linting is that I must be too used to how long Java parameters tend to be pretty long. Consider it fixed. :disappointed: